recipe/meta.yaml

```diff
@@ -36,12 +36,14 @@ requirements:
     - pyyaml >=5.3.1
     - toolz >=0.10.0
     - importlib-metadata >=4.13.0  # [py<312]
+    # ImportError: Missing optional dependency 'pyarrow'
+    - pyarrow >=14.0.1
   run_constrained:
     # array
     - numpy >=1.24
     # dataframe
```

Comment:
All dependencies needed for extras complete should be moved to run: https://github.com/dask/dask/blob/2025.5.1/pyproject.toml#L65-L69